nx-X11/extras/Mesa.patches/series

```diff
@@ -1,2 +1,4 @@
 4001_Arctica_Fix-non-working-GLX-in-64bit-Xorg-7.0.patch
```

Comment:
Drop the "Arctica" part from this patch name. Does not make sense, really. (I do stupid stuff sometimes, no need to copy me).